### ISSUE TITLE
Not able to trigger audio-caption "needs review"

### ISSUE THREAD

**Issue by dylanb** [dequelabs/axe-core#1856]
Expectation: `<audio>` elements without a `<track>` of the correct type should raise a needs review issue in the axe extension 

Actual: no issues are raised

Motivation: rules should work as advertised

<pre><code>
axe-core version: 3.3.2
Extension: 4.0.0
</code></pre>

Here is the document tried

```
<html lang="en">
    <head>
        <title>Audio caption failures</title>
    </head>
    <body>
        <main>
            <h1>Audio captions</h1>
            <p>This page is intended to test audio captions</p>
			<audio id="empty"></audio>
        </main>
    </body>
</html>
```

the element with `id="empty"` should raise a needs review item

**Comment by WilcoFiers** [comment]
This rule was deprecated in axe-core 3.1.0. Unlike video, the audio element does not support captions. Recommending people to add a feature that doesn't function seems like a waste of time. The correct way to make audio accessible is by providing a transcript. MDN recommends that if someone wants to use captions for audio, they use a video element:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio#Accessibility_concerns

Here's the PR in which it was removed: 
https://github.com/dequelabs/axe-core/pull/1071
